src/control/lib/control/http.go

```diff
@@ -113,9 +128,31 @@ func httpGetBodyRetry(ctx context.Context, req httpGetter) ([]byte, error) {
 	return result, err
 }
 
+// httpsGetFunc will prepare the GET requested using the certificate for secure mode
+// and return the http.Get
+func httpsGetFunc(cert []byte) (httpGetFn, error) {
+	caCertPool := x509.NewCertPool()
+	result := caCertPool.AppendCertsFromPEM(cert)
```

Comment:
We should be able to harvest what's in the OS list of trusted certs, similar to what web browsers do. I think `x509.SystemCertPool` does this.